zerver/lib/events.py

```diff
@@ -309,7 +309,9 @@ def fetch_initial_state_data(
         state["realm_icon_source"] = realm.icon_source
         add_realm_logo_fields(state, realm)
 
-        state["realm_uri"] = realm.url
+        # We keep the alias, realm_uri, for backwards-compatibility. One should use
+        # realm_url instead of the other realm_uri everywhere possible in the codebase.
```

Comment:
This is a comment that is reasonable for a commit message or changelog, but feels weird for the code itself -- because someone reading this in 5 years will read it as a description of the current code, rather than a description of a change. Better would be:

> # TODO/compatibility: realm_uri is a deprecated alias for realm_url that can be removed once there are no longer clients relying on it.